src/apps/concentrator/ethereum/concentrator.abc.token-fetcher.ts

```diff
@@ -35,7 +34,16 @@ export class EthereumCleverAbcTokenFetcher extends AppTokenTemplatePositionFetch
     ];
   }
 
-  async getPricePerShare() {
-    return [(await this.getContract(abcCVX).ratio()).div(10 ** 6).toNumber() / 2];
+  async getPricePerShare({ } : GetPricePerShareParams<AbcCvx> )  {
+    return [1]; 
```

Comment:
You can't have 2 underlying tokens and only 1 price per share value. Plus, it's not true the pricePerShare between this token and the underlying is 1 to 1.

Here's the formula for the pricePerShare 
`pricePerShare = reserves.map(r => r/supply)`

Reply:
Done!